Dockerfile

```diff
@@ -1,13 +1,32 @@
 FROM python:3.6
 
-# prepare directory
+# explicitly set user/group IDs
+RUN groupadd -r -g 999 kqueen && \
+    useradd -r -g kqueen -u 999 -m -s /sbin/nologin -c "Kqueen user" kqueen
+
+COPY . /code
+
 WORKDIR /code
 
-# copy app
-COPY . .
+RUN mkdir -p log/kqueen
 
 # install from local file
 RUN pip install .
 
-# run app
-CMD ./entrypoint.sh
+# grab gosu for easy step-down from root
```

Comment:
Why? You can use `USER` command IMO.

Reply:
official docker best practises provide several pros of gosu using, pls take a look https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
If you need `sudo` ...

We don't need sudo IMHO.

Reply:
yep, you are right, ill remove it